.github/workflows/dab-release.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: DAB requirements update
+on:
+  schedule:
+    - cron: '0 * * * *' # once an hour
+permissions:
+  pull-requests: write
+  contents: write
+jobs:
+  dab-pin-newest:
+    runs-on: ubuntu-latest
+    steps:
+      - id: dab-release
+        name: Get current django-ansible-base release version
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          owner: ansible
+          repo: django-ansible-base
+          excludes: prerelease, draft
+
+      - name: Check out respository code
+        uses: actions/checkout@v4
+
+      - id: dab-pinned
+        name: Get current django-ansible-base pinned version
+        run:
+          echo "version=$(requirements/django-ansible-base-pinned-version.sh)" >> "$GITHUB_OUTPUT"
+
+      - name: Update django-ansible-base pinned version to upstream release
+        run:
+          requirements/django-ansible-base-pinned-version.sh -s ${{ steps.dab-release.outputs.release }}
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v6
+        with:
+          base: devel
+          branch: bump-django-ansible-base
+          title: Bump django-ansible-base
```

Comment:
lets do this with version appended? unique branch name and unique PR title

Reply:
we use PR title to generate release note

Reply:
wonder how this action (as it's currently written) handles this following situation...
- we on dab x.y.z-1
- dab release x.y.z
- pr open to bump dab to x.y.z 
- we dont merge the pr
- dab release x.y.z+1
- (this automation runs)

would it update the branch (bump version and rebase from devel) since we reuse the branch name and update the PR?

Reply:
I believe it would update all the things ( branch updated to devel + dab x.y.z+1 + PR).

Best I can tell from the output messages and commands it's idempotent.